### PR TITLE
Add note about redirects

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3911

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ Additional site collections are now displayed. For example, all group and video
 The Detailed parameter has been deprecated. It will continue to work with earlier versions
 
 > [!NOTE]
-> Site collections in the Recycle Bin will not be retrieved by using the `Get-SPOSite` cmdlet.  
+> Site collections in the Recycle Bin will not be retrieved by using the `Get-SPOSite` cmdlet.
+> Sites redirects, like the ones created after [changing the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name) will be retrieved by using this cmdlet.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:54]:
```suggestion

> Sites redirects, like the ones created after [changing the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name) will be retrieved by using this cmdlet.
```
```suggestion

> Site redirects, like the ones created after [changing the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name) will be retrieved by using this cmdlet.
```

Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:54]:
```suggestion

> Site redirects, like the ones created after [changing the SharePoint domain name](/sharepoint/change-your-sharepoint-domain-name) will be retrieved by using this cmdlet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md [hunk 2]
```diff
@@ -50,7 +50,9 @@ Additional site collections are now displayed. For example, all group and video
 The Detailed parameter has been deprecated. It will continue to work with earlier versions
 
 > [!NOTE]
-> Site collections in the Recycle Bin will not be retrieved by using the `Get-SPOSite` cmdlet.  
+> Site collections in the Recycle Bin will not be retrieved by using the `Get-SPOSite` cmdlet.
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:54]:
```suggestion
Apologies @get-itips, missed it.
```

Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md:54]:
```suggestion
>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @get-itips, missed it.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**